docs/batch_inference_with_ts.md

```diff
@@ -129,7 +129,7 @@ $ curl localhost:8081/models/resnet-152
   * Get an image to test this service
 
     ```text
-    $ curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
+    $ curl -O https://raw.github.com/pytorch/serve/images/kitten.jpg
```

Comment:
need to include the image with this PR?
otherwise this url won't work.

Reply:
please also change s3 image path to github path for https://pytorch.org/serve/inference_api.html